azkaban-common/src/test/java/azkaban/trigger/BasicTimeCheckerTest.java

```diff
@@ -77,6 +77,29 @@ public void basicTimerTest() {
     }
 
     assertTrue(cond.isMet());
+  }
+
+  @Test
```

Comment:
Thanks for adding tests.

Could you at least add tests for DST handling? This will serve as documentation as well.